src/common/transformations/tests/utils/convert_precision.cpp

```diff
@@ -671,6 +671,14 @@ TEST(TransformationTests, ConvertPrecision_LogicalNot) {
 
     ASSERT_FALSE(has_type<element::Type_t::boolean>(f));
     ASSERT_TRUE(has_type<element::Type_t::u8>(f));
+
+    std::shared_ptr<ov::op::TypeRelaxed<opset4::LogicalNot>> tr;
+    for (const auto& node : f->get_ordered_ops())
+        if (auto op = std::dynamic_pointer_cast<ov::op::TypeRelaxed<opset4::LogicalNot>>(node))
```

Comment:
please, use ngraph RTTI to fix issue on macOS

Reply:
Unfortunately, it is not an option for TypeRelaxed -- I changed casting to TypeRelaxedBase which has no relation to Op / Node, etc. Hope it will work -- I saw numerous dynamic pointer casting to it in our code base.